app/views/home/how_government_works.html.erb

```diff
@@ -28,7 +28,7 @@
         <div class="one-half">
           <div class="inner-block">
             <h3>The Prime Minister</h3>
-            <p>The <a href="/government/ministers/prime-minister">Prime Minister</a> is head of the UK government. They are ultimately responsible for all policy and decisions. They:</p>
+            <p>The <a href="/government/ministers/prime-minister">Prime Minister</a> is the leader of Her Majesty’s Government. They are ultimately responsible for all policy and decisions. They:</p>
```

Comment:
They is plural and the Prime Minister is singular.
What about: 

The Prime Minister is the leader of Her Majesty’s Government and is ultimately responsible for all policy and decisions.

Reply:
Hi @davidlstanton, I'll pass your comment back to content design, but will merge this for now as this was the approved text. 👍